java/build.sh

```diff
@@ -9,7 +9,7 @@ mkdir -p target/classes/META-INF/lib
 
 swig -c++ -java -package org.rismch.verovio -outdir src/main/java/org/rismch/verovio verovio.i
 
-SRCFILES=`LS ../src/*.cpp`
+SRCFILES=`ls ../src/*.cpp`
```

Comment:
Using \ls avoids using aliases
Another way is to call /usr/bin/ls and in that case the shell won't replace it either.

Let me know which one is the most suitable and I'll propose and updated patch.
The problem is that currently LS is just not working at all :-(